airbyte-integrations/connectors/source-posthog/source_posthog/manifest.yaml

```diff
@@ -103,7 +103,7 @@ definitions:
       type: CustomIncrementalSync
       class_name: source_posthog.components.EventsCartesianProductStreamSlicer
       cursor_field: timestamp
-      partition_routers:
+      stream_slicers:
```

Comment:
This one is a bit gross, but because it overrides the CartesianProductStreamSlicer (which shouldn't even be in the language anymore), our underlying implementation detail (namely the `stream_slicers` field) bleeds into the code a little bit.